cypress/integration/granules_spec.js

```diff
@@ -107,20 +107,25 @@ describe('Dashboard Granules Page', () => {
     it('should be able to sort table by granule name', () => {
       cy.visit('/granules');
 
-      // save the name of the first granule
-      let firstGranuleName;
-      cy.get('.table .tbody .tr').eq(0).children('.td').eq(2).invoke('text')
-        .then((name) => (firstGranuleName = name));
+      // save the names of the granules
+      const originalGranuleNames = [];
+      cy.get('.table .tbody .tr')
+        .each(($row, index, $list) => {
+          cy.wrap($row).children('.td').eq(2).invoke('text').then((name) =>
+            originalGranuleNames.push(name));
+        });
 
       cy.get('.table .thead .tr .th').contains('.table__sort', 'Name').click();
       cy.get('.table .thead .tr .th').contains('.table__sort--asc', 'Name');
 
-      // wait until the name of the first granule changes
+      // wait until the names of the first two granules change
       cy.waitUntil(
         () => cy.get('.table .tbody .tr').eq(0).children('.td').eq(2).invoke('text')
-          .then((name) => name !== firstGranuleName),
+          .then((name) => name !== originalGranuleNames[0]) ||
+          cy.get('.table .tbody .tr').eq(1).children('.td').eq(2).invoke('text')
+            .then((name) => name !== originalGranuleNames[1]),
         {
-          timeout: 5000,
+          timeout: 10000,
```

Comment:
Do you know why you had to increase the timeout here?  I don't love that.

Reply:
I had failure here once.